controlpanel/frontend/jinja2/base.html

```diff
@@ -151,7 +151,16 @@
 
     <script src="{{ static('app.js') }}"></script>
     <script>window.moj.init();</script>
-
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-151666116-2"></script>
```

Comment:
You should remove the Google Analytics ID from here (and below) and use a template variable, taking the value from the Django settings module instead, eg:
```
<script async src="https://.../js?id={{ google_analytics_id }}"></script>
```
and in `controlpanel/frontend/jinja.py` add an item to the global template environment, eg:
```python
env.globals.update(
    {
        ...
        "google_analytics_id": settings.GOOGLE_ANALYTICS_ID,
    }
)
```
and in `controlpanel/settings/common.py`:
```python
...
GOOGLE_ANALYTICS_ID = "UA-..."
```
Not sure how sensitive the ID is, you might want to get a new one now that this has been published to Github.

Reply:
I don't think the ID is sensitive, so I don't think we need to get a new one. We can add filters in our Google Analytics account to ensure it only tracks traffic on our domains, in case someone finds it and tries to misuse it. Even if it is set as an environment variable, would it not be exposed in the source code anyway?

I've created different tracking IDs for alpha, prod and dev (old and new together). Not sure if I've implemented this in the right way though.